content/blog/open-source-monitoring-at-the-hpe-customer-technology-center-böblingen.md

```diff
@@ -0,0 +1,377 @@
xSeb/k29nrk7iH9DYHIjMgK1F5kvKfzTuYJfph05KAYbtST0G9kXGlpMN6S7hILHGjdhKZFv1kvNU02mOn2v6P4GGqCx4aVMgMgM2F1mQzfWOS80vi318xTlCrYWKTK3kCYfwiyyydLVfkW3eS85TQ6fpnni6Ow9PdeY5T8pnGyAyAzYXGdcalxKZJ/6FliihREMisnHhDQN+6bDIZJceRFZPLznP9p0mPfPJixD0rEdcnfGiaSUcAURmwPYiG9WaW451JVXHf8YlOog3DOU5TPV/4eWAyNJdwkH4UFasJrI6e8l5tuw0029htQKRGVCByHpgnWRse0mx/+Q7gltf4PrfpNNswWMQmQUQmQlLJ2PfS7ofE/srnNzF17q/6sPP8pYRAN49exCIzACIzITlklm8lyRn+Sab3RW6cbdyzq8hWbbgjg07zToLGCIzACIzYaFkVvi5JL9YGHU2iIzv09hv5mzSaUJkKiCyk8U8mXV+LimWcuJKOP+/34u+hp8eW7N+pwmRqYDIThbDZBbvJSPDi2MpQ2u5uMgCa3aaEJkKiKwE7ml0LDe5lHDw6Smsklmhl2ToKxhIVLW6yALrdJoQmQqIrBy6YpaTzgSTcy0lsnV6yZpZodOEyFScnMicGeTXsy/5ZRb//Xv4fh+u99HMYADngslvgI/2kl3cfC+OwweJDPcNfjL9P54vU8NE2sjr43MNNyBuUH4fzyOPRE4Ruc8Vslov2QSLdpoQmYpTFBlfz84SfomG3HuRRCt8xbuZIpCwELbQajQCDaR7OTHGTWHy/kGidZKjEHzOvZMms5JFdwz3e+bDeliM/wthnzA83Xj0FULX7YMHD9boJZuCOs1f/OIXv/71r+kGzRBSJH3P4HNUA0RmwH1FxpfzsERe2O3kBRZWSPzPEeUTXDA6xGiv9Chh8uGDDDdLkWiZ6fzhXMN+HifNybnkIO4oPM2fZbKVOVpkdN9BO9Lt2Nu3b3no5KE708ePH//pT3/6+eefSfS2VNi5Q2QGLCEyd0cWL3O+4qdOkS2jDaO99uWyd5DhrCPigabzD5/LkWwh3Mb019h5Szhb2HOy//EiC4RL99tvvz3ZF8gCHz9+PEGtQ2QGLCIy9/5wvofh633kkLDELggzZYihLSO5+MmHD8IMc8Nk+n9v/uFz0UDye+YOt03WZWL43fVw8PDvwD1FFvjxxx+p99E/HVvgM/1/kt0ohHvSwpe/CBCZAfcS2dYMd0vp4hZYJVP9Lcno28to5R6c+A0pRGZA0yIb7pZs7w3KsU3G7MIm08jPbYNyaCSsEi6y4X7TfaLlsJxM9MnGbf6DkuxEdpq95ASIzIC2RVYNSyRj0Go5G7Hhp82wl1zyZpPxTa3fGBbd8PCiJx3Gz2LPjQ9ZyMn2khMgMgMgMhMWSua+NyzJLVNcZAN5nIZSIw3LcnPmSW7DksPcQ2L44UYKRGYARGbCoskcf9lH7RB8Rzbcdo1UlEz0y+fJe/iI4XYu3pEdC3rJfSAyAyAyE1ZI5phGjJzEN1XSBcq9FslKfhuDGC174/GKf8lMDhTfl3KU0NBLHgQiM2BdkQ3f2RfjPqdI70vKWEFkRPHtzPFfkDHoJWeAyAxYV2QrkBMZXdOfFFztIgsUSKECkaGX/CQ1ioyeM/5ViEZ48uTJiiILlqF/A6OLTEbTGSwlZyH59e/YA4W9001uZBAZbQnDNJCeMh33E+V0yYvahayv+CbaNPSSGqoTGX3P4V9ObQcS2YcPH/gL+BQ2Ijv43tXBP345zuBh556w2TnHL9GQnzRsikeX7TxM40FfB8flHH5g7FY964uMqPlmB72knupE1j02IgvyIaJQCBoRhRxYHuwzLLGGktl+Mb7liU6VED4D6ND43gH4zGVsIrJAbcpAL1kKRLY2ViILjLSRbkmWeTFaKF3aF5mfHHdOD+iI86bj6QgttSeyQCVNHHrJI4DI1sZEZJMf8wu0TUZlmafMi4ynhskjLUXczLBKS5Px5BDDZ16UsrnIiG1vhdBLHg1EtjY1XK5T4p3WttSTzPpCQS95TyCytYHIctSWzGotHnrJ+wORrU2NIquDCpNZ+kYJvaQVENnaQGQ5qk1mCd2gl7QFIlsbiCxH5ckYNoDoJc2ByNYGIstRfzL3v41CL7kQENnaQGQ5WknmOBmhl1wUiGxtILIcbSVT1B6il1waiGxtILIczSWjuclCL7kOENnaQGQ5Gk0mpyr0kmsCka0NRJaj6WQmzSN6yZWByNaGipu+gYfPj9yc77777quvvuKVrXn27FnTipdbMPSS6wORrQ3VN1+4FfD06dN6rEp00IX9/e9///e//80rYC0gspMGfa45L168ePnyJa+AtYDIThqIzJyHDx9Sd8krYC0gspMGIrPlzZs34VPZ9B99DkyAyE4aiMwW6iuDyNBdrgxEdtJAZLZQXxlEhu5yZSCykwYiM0T6ygC6yzWByE4aiMwQ6SsD6C7XBCI7aSAyQ6SvDKC7XBOI7KSByKyY9JUBdJerAZGdNBCZFZO+MoDucjUgspMGIrNi0lcG0F2uBkR20kBkJhzsKwPoLtcBIjtpIDJzKE9KlVfAWkBkJw1EZg5EtgkQ2UkDkZkDkW0CRHbSQGTmQGSbAJGdNBCZORDZJkBkJw1EZg5EtgkQ2UkDkZkDkW0CRHbSQGTmQGSbAJGdNBCZORDZJkBkJw1EZg5EtgkQ2UkDkZkDkW0CRHbSQGTmQGSbAJGdNBCZORDZJkBkJw1EZg5EtgkQ2UkDkZkDkW0CRHbSQGTmQGSbAJGdNBCZORDZJkBkJw1EZg5EtgkQ2UkDkZkDkW0CRHbSQGTmQGSbAJGdNBCZORDZJkBkJw1EZg5EtgkQ2UkDkZkDkW0CRHbSQGTmQGSbAJGdNK9fv+Y/JAvsoFQ5X7AWEBkAoHH++9//Dz4D97xiqIjiAAAAAElFTkSuQmCC "ILO Prometheus Connector - Architecture")
```

Comment:
Hmmm... not sure what this is. I think it's supposed to be a picture, but it's not rendering properly in the Preview:

![image](https://github.com/hpe-dev-incubator/hpe-dev-portal/assets/77496261/c02f5a6a-d8a6-482a-a7a8-d385a511741b)